contents/docs/libraries/ios/index.mdx

```diff
@@ -36,6 +36,18 @@ PostHog supports the following Apple platforms:
 
 <IOSSendEvents />
 
+## Preventing sensitive data capture
+
+To exclude specific UI elements from Autocapture, add `ph-no-capture` as either an `accessibilityLabel` or `accessibilityIdentifier`. When Autocapture detects this label or identifier anywhere in the view hierarchy, the event will be ignored:
+
+```swift
+// This view will be excluded from Autocapture
+let view = UIView()
+view.accessibilityLabel = "ph-no-capture"
+```
+
+> **Important:** By default, PostHog automatically excludes fields detected as sensitive, even without the `ph-no-capture` tag. These include password fields, credit card fields, OTP fields, and any other fields related to Personally Identifiable Information (PII).
```

Comment:
does the autocapture reuse the checks from session replay? just to be sure this is here in this section and it works for `captureElementInteractions` as well

Reply:
we already have posthog.com/contents/docs/session-replay/_snippets/ios-privacy.mdx btw, maybe consider improving that area and linking it back here if possible?
i think there are other areas of the docs mentioning about the ph no capture and what is automatically masked already